test/helpers/pagelets_helper_test.rb

```diff
@@ -1,17 +1,17 @@
 require 'test_helper'
+require 'statics_isolator_helper'
 
 class PageletsHelperTest < ActionView::TestCase
+  include Isolator
   include PageletsHelper
 
+  isolate_config Pagelets::Manager
```

Comment:
The proble I was trying to solve, is preservation of pagelets' registry between tests - The original code had `Pagelets.Manager.clear` in the teardown, which removed pagelets registered outside the tests (my pagelets). The isolator remembers the state of the local registry, and restores it after the test is done.